src/libs/ReportActionsUtils.ts

```diff
@@ -1664,6 +1678,42 @@ function getRemovedFromApprovalChainMessage(reportAction: OnyxEntry<ReportAction
     return Localize.translateLocal('workspaceActions.removedFromApprovalWorkflow', {submittersNames});
 }
 
+function isCardIssuedAction(reportAction: OnyxEntry<ReportAction>) {
+    return isActionOfType(reportAction, CONST.REPORT.ACTIONS.TYPE.CARD_ISSUED, CONST.REPORT.ACTIONS.TYPE.CARD_ISSUED_VIRTUAL, CONST.REPORT.ACTIONS.TYPE.CARD_MISSING_ADDRESS);
+}
+
+function getCardIssuedMessage(reportAction: OnyxEntry<ReportAction>, shouldRenderHTML = false) {
+    const assigneeAccountID = (reportAction?.originalMessage as IssueNewCardOriginalMessage)?.assigneeAccountID;
+    const assigneeDetails = PersonalDetailsUtils.getPersonalDetailsByIDs([assigneeAccountID], currentUserAccountID ?? -1)[0];
+
+    const assignee = shouldRenderHTML ? `<mention-user accountID=${assigneeAccountID}></mention-user>` : assigneeDetails?.displayName ?? assigneeDetails.login ?? '';
+    const link = shouldRenderHTML
+        ? `<a href='${environmentURL}/${ROUTES.SETTINGS_WALLET}'>${Localize.translateLocal('cardPage.expensifyCard')}</a>`
+        : Localize.translateLocal('cardPage.expensifyCard');
```

Comment:
So for mentions I think we need to copy the mention, as well, to keep it same as with a normal message 

<img width="553" alt="image" src="https://github.com/user-attachments/assets/7581bfa8-f544-4d09-949b-810b6353a45c">

Similarly for the LHN preview, we should show the `@FirstName` format. so its not a full Display name for the mention but the first name part

Reply:
<img width="683" alt="Screenshot 2024-09-14 at 12 28 43" src="https://github.com/user-attachments/assets/9861dbf7-bcde-46ff-a54b-3db6d514aef4">

<img width="631" alt="Screenshot 2024-09-14 at 12 27 55" src="https://github.com/user-attachments/assets/2f6cd1bd-c00a-45f3-b848-0f6fff2471a6">

<img width="367" alt="Screenshot 2024-09-14 at 12 27 41" src="https://github.com/user-attachments/assets/25c85435-45bf-442e-93b3-80aa1277803e">

@mountiny fixed